.github/workflows/test.yaml

```diff
@@ -652,10 +652,10 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
-      - name: use node 12
+      - name: use node 14
```

Comment:
This is causing the Firebase tests to fail right now - we either need to fix these as well, or split apart (fixing preferred)